ui-core/src/components/inputs/Slider.tsx

```diff
@@ -38,7 +43,7 @@ const Slider = ({
   };
 
   return (
-    <div className={cx('range-wrapper', { disabled })} style={{ width: `${width}px` }}>
+    <div className={cx('range-wrapper', className, { disabled })} style={{ width: `${width}px` }}>
```

Comment:
Hm, it seems a bit arbitrary to apply part of the props to the wrapper `<div>` and the rest to the `<input>`… This is a bit surprising especially when the `className` field comes from the `HTMLInputElement` type. And it seems like this isn't used?

Reply:
The container allows to have a larger region to click on, and I need to way to access this container style to position it when I use it in OSRD.

We could offer two different props 
- `containerClassName` 
- className for the input itself

but we can always access with a selector from its parent

Reply:
> className for the input itself

That's the current behavior FWIW: `className` is currently part of `...rest` and forwarded to the input.

> containerClassName

Sounds fine to me: if we need to control the class name of the wrapper from the outside, I think a property named differently like this is better.

> but we can always access with a selector from its parent

Right, if this doesn't make our life terrible in OSRD, this sounds good as well (and we don't need the new prop).